cross-file/native-remote.ini

```diff
@@ -0,0 +1,24 @@
+# This a cross-file for the native probe, providing sane default options for it.
```

Comment:
I think all the ini files need a more verbose comment at the top of each file explaining what it's tradeoffs are, and that it belongs to a group of different firmware "options".

Reply:
Ah, good catch! Yes, that's a really good point.